From b998a0f37a8c06bc0a1cd63717c091991ee0f120 Mon Sep 17 00:00:00 2001
From: Allen Wittenauer <aw@apache.org>
Date: Sat, 28 Feb 2015 22:53:38 -0800
Subject: [PATCH 2350/2848] MAPREDUCE-5653. DistCp does not honour
 config-overrides for
 mapreduce.[map,reduce].memory.mb (Ratandeep Ratti
 via aw)

Change-Id: Ie866ef9764e763740767383e56968917ffc418a0
---
 .../src/main/resources/distcp-default.xml          |   10 ----------
 1 file changed, 10 deletions(-)

diff --git a/hadoop-tools/hadoop-distcp/src/main/resources/distcp-default.xml b/hadoop-tools/hadoop-distcp/src/main/resources/distcp-default.xml
index f50dddd..6e1154e 100644
--- a/hadoop-tools/hadoop-distcp/src/main/resources/distcp-default.xml
+++ b/hadoop-tools/hadoop-distcp/src/main/resources/distcp-default.xml
@@ -32,16 +32,6 @@
     </property>
 
     <property>
-        <name>mapred.job.map.memory.mb</name>
-        <value>1024</value>
-    </property>
-
-    <property>
-        <name>mapred.job.reduce.memory.mb</name>
-        <value>1024</value>
-    </property>
-
-    <property>
         <name>mapred.reducer.new-api</name>
         <value>true</value>
     </property>
-- 
1.7.9.5

